src/components/Header/TopMainNav/SearchBar/SearchDisplay.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { EventHandler, SyntheticEvent } from 'react';
+import cn from 'classnames';
+
+export const SearchDisplay = ({
+  isMobile = false,
+  onClick,
+  children,
+}: {
+  isMobile?: boolean;
+  onClick: EventHandler<SyntheticEvent<HTMLDivElement, MouseEvent>>;
+  children: React.ReactNode;
+}) => (
+  <div
+    onClick={onClick}
+    className={cn(
+      'h-48, px-16  bg-light-grey border border-mid-grey rounded-md flex-row justify-self-start self-center hover:bg-white hover:border-light-grey relative focus-within:bg-white focus-within:shadow-input focus-within:border-transparent focus-within:outline-gui-focus',
+      {
+        'md:hidden max-w-4/5 m-24 flex transition-input': isMobile,
+        'hidden sm:flex mx-16 flex-shrink hover:bg-white': !isMobile,
```

Comment:
Not a massive fan of this approach. I think it's best to either rely on CSS for mobile styling _or_ JS. Using both at the same time can lead to a lot of incidental complexity. Potentially we could use here non-tailwind CSS if needed?

Reply:
All I did here was combine 2 components that each already had this `isMobile` prop. Wasn't really thinking about that. Shall I spend more time and have a look how to do it with css only?

Reply:
Yeah, there seem to be a lot of things going on, I wouldn't wanna do it in this ticket. If we really wanna improve that, I suggest we create another ticket.

Reply:
OH actually, it's not a css thing. It's a `variant` kind of situation. We're using this SearchBar in 2 different components and we're not using any `useMediaQuery` hooks or anything. `isMobile` value is hardcoded for each specific case. So we're fine here, I think.